tests/CustomerBundle/DependencyInjection/SonataCustomerExtensionTest.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\CustomerBundle\Tests\DependencyInjection;
+
+use Matthias\SymfonyDependencyInjectionTest\PhpUnit\AbstractExtensionTestCase;
+use Sonata\CustomerBundle\DependencyInjection\Configuration;
+use Sonata\CustomerBundle\DependencyInjection\SonataCustomerExtension;
+use Symfony\Bundle\TwigBundle\DependencyInjection\TwigExtension;
+use Symfony\Component\Config\Definition\Processor;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface;
+
+final class SonataCustomerExtensionTest extends AbstractExtensionTestCase
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->setParameter('kernel.bundles', ['SonataCustomerBundle' => true]);
+    }
+
+    /**
+     * @group legacy
```

Comment:
Are the methods marked with "legacy" group intended to be kept in the next major release?

Reply:
Methods marked with "legacy" should be rewrite in next major. They are from older version (based on SonataUserBundle 3.x).

Reply:
Then, please add the corresponding `NEXT_MAJOR` comments.

Reply:
I update test. Now `NEXT_MAJOR` is not required.